index.html

```diff
@@ -4,12 +4,580 @@
+                        </figure>
+                    </a>
+                </li>
+            </ul>
+        </section>
+        <section class="featured-works">
+            <h2 class="fw-heading">Spotlight Digital Assets Selections</h2>
+            <button class="fw-button">Explore Featured Works</button>
+            <ul class="fw-list">
+                <li class="fw-list-item">
+                    <a href="#" class="fw-product">
+                        <figure class="fw-product-figure">
+                            <img src="#" alt="photo" class="fw-product-image">
+                            <figcaption class="fw-product-caption">
+                                <h3 class="fw-product-title">Flexsoul</h3>
+                                <p class="fw-product-price">$12.00</p>
+                                <p class="fw-prodcut-category">Yoga Studio</p>
+                                <p class="fw-product-sub">UI Kits</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+                <li class="fw-list-item">
+                    <a href="" class="fw-product">
+                        <figure class="fw-product-figure">
+                            <img src="#" alt="photo" class="fw-product-image">
+                            <figcaption class="fw-product-caption">
+                                <h3 class="fw-product-title">Avery</h3>
+                                <p class="fw-product-price">$6.00</p>
+                                <p class="fw-prodcut-category">Monthly Report</p>
+                                <p class="fw-product-sub">Presentation</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+                <li class="fw-list-item">
+                    <a href="" class="fw-product">
+                        <figure class="fw-product-figure">
+                            <img src="#" alt="photo" class="fw-product-image">
+                            <figcaption class="fw-product-caption">
+                                <h3 class="fw-product-title">Studio</h3>
+                                <p class="fw-product-price">$8.00</p>
+                                <p class="fw-prodcut-category">Business Proposal</p>
+                                <p class="fw-product-sub">Print Template</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+            </ul>
+        </section>
+        <section class="trending-products">
+            <h2 class="tp-heading">Trending Products</h2>
+            <button class="tp-button">Explore Trending Works</button>
+            <ul class="tp-list">
+                <li class="tp-list-item">
+                    <a href="#" class="tp-product">
+                        <figure class="tp-product-figure">

```

Comment:
The card elements in the previous block have the same design and content. They should use `<article> `and the same classes for consistency.